cmd/lockinfo-handlers_test.go

```diff
@@ -26,8 +26,11 @@ import (
 func TestListLocksInfo(t *testing.T) {
 	// Initialize globalNSMutex to validate listing of lock
 	// instrumentation information.
-	isDistXL := false
-	initNSLock(isDistXL)
+	adminTestBed, err := prepareAdminXLTestBed()
```

Comment:
Is admin xl test bed required to test `ListLockInfo()` or just XL layer is enough?

Reply:
XL Layer and initNSLocks is required. Using admin xl test bed because it does both

Reply:
If `prepareAdminXLTestBed()` does more (or less) in future, its not a good idea to fail this test.  IMO its better to init all required things to this test file (or layer)

Reply:
list locks functionality is used only by mc admin AFAIK. Given that, using prepareAdminXLTestBed() should be okay to avoid code duplication. Agree?

Reply:
Do you agree we need all the things?
```go
// prepareAdminXLTestBed - helper function that setups a single-node
// XL backend for admin-handler tests.
func prepareAdminXLTestBed() (*adminXLTestBed, error) {
	// reset global variables to start afresh.
	resetTestGlobals()

	// Initialize minio server config.
	rootPath, err := newTestConfig(globalMinioDefaultRegion)
	if err != nil {
		return nil, err
	}
	// Initializing objectLayer for HealFormatHandler.
	objLayer, xlDirs, xlErr := initTestXLObjLayer()
	if xlErr != nil {
		return nil, xlErr
	}

	// Initialize boot time
	globalBootTime = UTCNow()

	globalEndpoints = mustGetNewEndpointList(xlDirs...)

	// Set globalIsXL to indicate that the setup uses an erasure code backend.
	globalIsXL = true

	// initialize NSLock.
	isDistXL := false
	initNSLock(isDistXL)

	// Setup admin mgmt REST API handlers.
	adminRouter := router.NewRouter()
	registerAdminRouter(adminRouter)

	return &adminXLTestBed{
		configPath: rootPath,
		xlDirs:     xlDirs,
		objLayer:   objLayer,
		mux:        adminRouter,
	}, nil
}
```